src/etl/allele_etl.py

```diff
@@ -151,267 +156,313 @@ class AlleleETL(ETL):
+        query_and_file_list = self.process_query_params(query_template_list)
         CSVTransactor.save_file_static(generators, query_and_file_list)
         Neo4jTransactor.execute_query_batch(query_and_file_list)
 
     def get_generators(self, allele_data, data_provider, batch_size):
+        '''Create Generators'''
 
-        dataProviders = []
+        data_providers = []
         release = ""
         alleles_no_constrcut_no_gene = []
         alleles_construct_gene = []
         alleles_no_construct = []
-        alleles_no_gene =[]
+        alleles_no_gene = []
         allele_synonyms = []
-        allele_secondaryIds = []
-        crossReferenceList = []
+        allele_secondary_ids = []
+        cross_reference_list = []
 
         counter = 0
-        dateProduced = allele_data['metaData']['dateProduced']
+        date_produced = allele_data['metaData']['dateProduced']
 
-        dataProviderObject = allele_data['metaData']['dataProvider']
+        data_provider_object = allele_data['metaData']['dataProvider']
 
-        dataProviderCrossRef = dataProviderObject.get('crossReference')
-        dataProvider = dataProviderCrossRef.get('id')
-        dataProviderPages = dataProviderCrossRef.get('pages')
-        dataProviderCrossRefSet = []
+        data_provider_cross_ref = data_provider_object.get('crossReference')
+        data_provider = data_provider_cross_ref.get('id')
+        data_provider_pages = data_provider_cross_ref.get('pages')
+        data_provider_cross_ref_set = []
 
-        loadKey = dateProduced + dataProvider + "_ALLELE"
+        load_key = date_produced + data_provider + "_ALLELE"
 
         #TODO: get SGD to fix their files.
 
-        if dataProviderPages is not None:
-            for dataProviderPage in dataProviderPages:
-                crossRefCompleteUrl = ETLHelper.get_page_complete_url(dataProvider, self.xrefUrlMap, dataProvider, dataProviderPage)
-
-                dataProviderCrossRefSet.append(ETLHelper.get_xref_dict(dataProvider, dataProvider, dataProviderPage,
-                                                                             dataProviderPage, dataProvider, crossRefCompleteUrl,
-                                                                             dataProvider + dataProviderPage))
-
-                dataProviders.append(dataProvider)
-                logger.info("data provider: " + dataProvider)
+        if data_provider_pages is not None:
+            for data_provider_page in data_provider_pages:
+                cross_ref_complete_url = ETLHelper.get_page_complete_url(data_provider,
+                                                                         self.xref_url_map,
+                                                                         data_provider,
+                                                                         data_provider_page)
+
+                data_provider_cross_ref_set.append(ETLHelper.get_xref_dict(\
```

Comment:
can we include the first item in the list on the same line as the function call - I think that makes it more readable.